stock_production_lot_multi_company/views/stock_production_lot_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="stock_production_lot_company_form" model="ir.ui.view">
```

Comment:
The record `id` should be `view_production_lot_form`  see the [OCA Guidelines on inherited xml](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#inherited-xml)